acceptance_tests/features/pages/sign_in_internal.py

```diff
@@ -22,5 +22,17 @@ def enter_incorrect_password():
     browser.driver.find_element_by_id('password').send_keys('words')
 
 
-def internal_sign_in_btn():
+def internal_sign_in_button():
```

Comment:
Shouldn't this be click_internal_sign_in_button()?  Other entry fields are descriptive of their actions (e.g., enter_incorrect_username()) but this one isn't